src/components/reviews/CreateReview.jsx

```diff
@@ -83,7 +196,7 @@ class CreateReview extends React.Component {
         <form onSubmit={this.onSubmit} onChange={this.onChange}>
           <h2>Write a new review</h2>
           <label>Overall rating: *</label>
-          <StarInput rating={rating}/>
+          <StarInput rating={rating} setRating={this.setRating}/>
           <br></br>
```

Comment:
Do you need all the breaks `<br>` here? Seems to bloat your HTML- if you need spacing can you achieve that with some padding properties on these form elements?

Reply:
There are a lot of different-sized spacings in the component so I felt that I would rather bloat the HTML a little bit than add 5-6 more classes to the already messy CSS.

Reply:
Ok. It works for now-- but if we are addressing style later I think we will likely want the ability to modify the space between these elements. 

What if you made a `<div>` with class name of `.spacer` or similar and put that in place of the `<br>`s? This way we can modify the .spacer height appropriately without having to also edit your CreateReview component.